File: aifupan_project/ReviewAnalysis/vo/ai/DanMuVo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ReviewAnalysis.vo.ai
{
    public class DanMuVo
    {
        /* 原Java中被注释的字段（按需取消注释）
        /// <summary>租户ID</summary>
        public long TenantId { get; set; }

        /// <summary>主播ID</summary>
        public string SecUid { get; set; }

        /// <summary>消息ID</summary>
        public string MsgId { get; set; }

        /// <summary>录制用户ID</summary>
        public long UserId { get; set; }

        /// <summary>视频ID</summary>
        public string VideoId { get; set; }
        */

        /// <summary>记录时间戳（Unix毫秒）</summary>
        public long? recordDate { get; set; }

        /// <summary>直播场次唯一标识号</summary>
        public string batchNumber { get; set; }

        /// <summary>发送弹幕的用户昵称</summary>
        public string nickName { get; set; }

        /// <summary>标识是否为新用户（首次发言）</summary>
        public bool? isNew { get; set; }

        /// <summary>用户等级（平台等级体系）</summary>
        public long? level { get; set; }

        /// <summary>本场直播中用户的最小粉丝团等级</summary>
        public long? fansLevelMin { get; set; }

        /// <summary>本场直播中用户的最大粉丝团等级</summary>
        public long? fansLevelMax { get; set; }

        /// <summary>用户当前的粉丝团等级</summary>
        public long? fansLevelCurrent { get; set; }

        /// <summary>弹幕文本内容（需注意敏感词过滤）</summary>
        public string content { get; set; }

        /// <summary>本场直播中该用户的弹幕发送总次数</summary>
        public int? countSendNum { get; set; }

        /// <summary>弹幕排序权重值（越大优先级越高）</summary>
        public long? sort { get; set; }

        /// <summary>标记是否已向用户显示等级提升提示（默认false）</summary>
        public bool? showUpgradeTips { get; set; } = false;

        /// <summary>标识内容是否已进行脱敏处理（如昵称打码）</summary>
        public bool? isDesensitization { get; set; }
    }
}

File: aifupan_project/fupan-server/replay-third/src/main/java/com/jiuyu/replay/third/vo/DanMuVo.java
package com.jiuyu.replay.third.vo;

/**
 * @author ：lujie
 * @description：
 * @date ：2025/1/17 下午8:21
 */

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.io.Serializable;

/**
 * 弹幕数据
 */
@Data
@Schema(description = "弹幕返回详细参数")
public class DanMuVo implements Serializable {
    private static final long serialVersionUID = 1L;

//    @Schema(description = "租户id")
//    private Long tenantId;
//
//    @Schema(description = "主播id")
//    private String secUid;
//
//    @Schema(description = "消息id")
//    private String msgId;
//
//    @Schema(description = "录制用户id")
//    private Long userId;
//
//    @Schema(description = "视频id")
//    private String videoId;

    @Schema(description = "记录时间戳")
    private Long recordDate;

    @Schema(description = "直播场次号")
    private String batchNumber;

    @Schema(description = "弹幕用户昵称")
    private String nickName;

    @Schema(description = "是否新用户")
    private Boolean isNew;

    @Schema(description = "弹幕用户等级")
    private Long level;

    @Schema(description = "场次中最小粉丝团等级")
    private Long fansLevelMin;

    @Schema(description = "场次中最大粉丝团等级")
    private Long fansLevelMax;

    @Schema(description = "当前粉丝团等级")
    private Long fansLevelCurrent;

    @Schema(description = "弹幕内容")
    private String content;

    @Schema(description = "弹幕发送次数")
    private Integer countSendNum;

    @Schema(description = "弹幕排序")
    private Long sort;

    @Schema(description = "在之前是否显示升级提示")
    private Boolean showUpgradeTips = false;

    @Schema(description = "是否脱敏")
    private Boolean isDesensitization;

    @Schema(description = "是否是福袋弹幕字段")
    private Boolean isBlessBag;
}